optimum/onnxruntime/modeling_ort.py

```diff
@@ -159,6 +185,43 @@ def _save_pretrained(self, save_directory: Union[str, Path], file_name: Optional
         dst_path = Path(save_directory).joinpath(model_file_name)
         shutil.copyfile(src_path, dst_path)
 
+    @classmethod
+    @add_start_docstrings(FROM_PRETRAINED_START_DOCSTRING)
```

Comment:
why did you add this method? For documentation reasons?
Nice idea with the `add_start_docstrings`!

Reply:
I was thinking it makes sense to have `from_pretrained` in the `ORTModel` section of the documentation. If in the future we extend these pipeline wrappers to other backends (say, fx or OpenVino), they may have custom parameters that the documentation from `OptimizedModel` will not capture. So It makes sense to me to have the doc for `from_pretrained` in the subclasses as well.

What do you think?